SYCL/Matrix/joint_matrix_all_sizes.cpp

```diff
@@ -0,0 +1,25 @@
+//==-------- joint_matrix_all_sizes.cpp  - DPC++ joint_matrix---------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: matrix
+
+// RUN: %clangxx -fsycl %s -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+// XFAIL: gpu
```

Comment:
could you please clarify what is the reason for xfail on gpu? Is there lack of support?

Reply:
Yes, there is lack of support of these sizes on the GPU